import-service/src/handlers/importFileParser/handler.ts

```diff
@@ -0,0 +1,82 @@
+import { S3Event, S3EventRecord } from "aws-lambda";
+import { SQSClient, SendMessageCommand } from "@aws-sdk/client-sqs";
+import { S3Client, GetObjectCommand, CopyObjectCommand, DeleteObjectCommand } from "@aws-sdk/client-s3";
+import csv from "csv-parser";
+import { Readable, Writable } from "stream";
+import { pipeline }  from "stream/promises";
+import { queueUrl } from "../../constants";
+
+const s3 = new S3Client();
+const sqs = new SQSClient();
+
+const handleRecord = async (record: S3EventRecord) => {
+    const bucketName = record.s3.bucket.name;
+    const key = record.s3.object.key;
+    const s3Command = new GetObjectCommand({
+        Bucket: bucketName,
+        Key: key,
+    });
+    const s3Object = await s3.send(s3Command);
+
+    if (!s3Object.Body) {
+        throw new Error("Object body does not exist");
+    };
+
+    if (!(s3Object.Body instanceof Readable)) {
+        throw new Error("Object body is not Readable");
+    };
+
+    const stream = s3Object.Body;
+    await pipeline(
+        stream,
+        csv(),
+        new Writable({
+            objectMode: true,
+            write: async (chunk, encoding, done) => {
+                chunk.price = Number(chunk.price);
```

Comment:
It's a bad practice in JS to mutate properties of original object (so we are breaking the original object and debugging can be difficult), better create new one

```
  const sqsCommand = new SendMessageCommand({
    QueueUrl: queueUrl,
    MessageBody: JSON.stringify({ ...chunk, price: Number(chunk.price), count: Number(chunk.count) }),
  });
```

Reply:
done